packages/typechain/src/utils/signatures.ts

```diff
@@ -13,6 +14,11 @@ export function getIndexedSignatureForEvent(event: EventDeclaration): string {
   return `${event.name}(${indexedType.map((e) => e.type.originalType).join(',')})`
 }
 
+export function getArgumentSignature(argument: AbiParameter): string {
```

Comment:
just a naming nitpick -- the whole string returned by `getSignatureForFn` is a function signature, a single input type is not called a signature :)
```suggestion
export function getArgumentForSignature(argument: AbiParameter): string {
```